client/src/document/on-github.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+
+import { Doc } from "./types";
+
+export function OnGitHubLink({ doc }: { doc: Doc }) {
+  return (
+    <div id="on-github">
+      <h4>Found a problem with this page?</h4>
+      <ul>
+        <li>
+          <SourceOnGitHubLink doc={doc} />
+        </li>
+        <li>
+          <NewIssueOnGitHubLink doc={doc} />
+        </li>
+      </ul>
+    </div>
+  );
+}
+function SourceOnGitHubLink({ doc }: { doc: Doc }) {
+  const { github_url, folder } = doc.source;
+  return (
+    <a
+      href={github_url}
+      title={`Folder: ${folder}`}
```

Comment:
For people using assistive technologies, this will read:

"Source on Github. Folder: /content/HTML"

for example. Perhaps that is totally fine but, thought I would just mention it in case we can word it differently. Here I would also be explicit about the fact that it will open in a new tab:

```suggestion
      title={`Folder: ${folder} (Opens in new tab)`}
```